presto-tests/src/test/java/com/facebook/presto/memory/TestClusterMemoryLeakDetector.java

```diff
@@ -46,49 +47,27 @@ public void testLeakDetector()
         QueryId testQuery = new QueryId("test");
         ClusterMemoryLeakDetector leakDetector = new ClusterMemoryLeakDetector();
 
-        leakDetector.checkForMemoryLeaks(() -> ImmutableList.of(), ImmutableMap.of());
+        leakDetector.checkForMemoryLeaks(Collections.emptyMap(), ImmutableMap.of());
         assertEquals(leakDetector.getNumberOfLeakedQueries(), 0);
 
-        // the leak detector should report no leaked queries as the query is still running
```

Comment:
Why is this test being removed?

Reply:
Because we removed the method checkForClusterMemoryLeaks and thus the unit test method around that. checkForClusterMemoryLeaks was a "fork (copy/paste)" of checkForMemoryLeaks. I unified the two.